style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+* *::before *::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  margin: 10px;
+  padding: 10px;
+  border: 20px;
+  background-image: url(Header.svg);
+  max-height: 100vh;
+  max-width: auto;
+}
+
+a {
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 10px;
+  letter-spacing: 2px;
+  text-decoration: none;
+  background: none;
+  padding: 5px 5px;
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+}
+
+a:hover {
+  background-color: #6070ff;
+}
+
+a:enabled {
+  background-color: #fff;
+  color: #7f8cff;
+}
+
+a:disabled {
+  background-color: #5e6c84;
+}
+
+a:active {
+  background: #2230d2;
+}
+
+.toggleNav {
+  display: none;
+}
```

Comment:
- [ ] The hamburger menu 🍔 is not displaying on the **mobile screen**, it should be showing instead of the nav links, Kindly display the button and hide the nav links instead.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/219694211-763bd0dd-20b8-4860-841a-3a33e6460d0a.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219694393-321433b4-b274-4b43-9ff4-573fc03919f2.png)